_wiki/Install/Linux.md

```diff
@@ -24,13 +23,7 @@ systemctl --user daemon-reload
 systemctl --user enable opentabletdriver --now
 ```
 
-<small class="text-muted">
-Note: If you have the Microsoft dotnet repository installed you will have to either make sure you are
-not using any packages from that repository or use everything dotnet based off that. Mixing packages
-from different repositories will cause libhostfxr issues.\\
-If you are experiencing these see these solutions from Microsoft
-[here](https://learn.microsoft.com/en-us/dotnet/core/install/linux-package-mixup#solutions).
-</small>
+> Note: If you have the Microsoft dotnet repository installed you will have to either make sure you are not using any packages from that repository or use everything dotnet based off that. Mixing packages from different repositories will cause libhostfxr issues. If you are experiencing these see these solutions from Microsoft [here](https://learn.microsoft.com/en-us/dotnet/core/install/linux-package-mixup#solutions).
```

Comment:
This is a bit run-on, also causing some small text flow changes.
Pre-PR:
![image](https://github.com/OpenTabletDriver/opentabletdriver.github.io/assets/421345/120fe72a-7d56-44d7-ba33-c962d8c0d4aa)

Post-PR:
![image](https://github.com/OpenTabletDriver/opentabletdriver.github.io/assets/421345/ff147184-6dab-4d0f-9ab3-5798f1500e4b)

The following change should keep the newline. I added 2 newlines for Markdown readability, but the first newline does not propagate into the displayed HTML.

```suggestion
> Note: If you have the Microsoft dotnet repository installed you will have to either make sure you are not using any packages from that repository or use everything dotnet based off that.
> Mixing packages from different repositories will cause libhostfxr issues.\\
> If you are experiencing these see these solutions from Microsoft [here](https://learn.microsoft.com/en-us/dotnet/core/install/linux-package-mixup#solutions).
```

Reply:
I kind of prefer to have these blockquotes be just one big block, no newlines. If it's too long that it needs a newline then we probably should not blockquote it.

Reply:
On the old C# site we do what you're saying so I'll just take it as it is:
![image](https://github.com/OpenTabletDriver/opentabletdriver.github.io/assets/421345/b62a10a1-5c41-4633-bae6-6ac825216517)